test/letters/containers/VeteranBenefitSummaryLetter.unit.spec.jsx

```diff
@@ -39,11 +42,39 @@ describe('<VeteranBenefitSummaryLetter>', () => {
 
   it('should show benefit info options', () => {
     const tree = SkinDeep.shallowRender(<VeteranBenefitSummaryLetter store={store} {...defaultProps}/>);
-    expect(tree.subTree('hasAdaptedHousing')).to.exist;
+    const rows = tree.dive(['div', '#benefitInfoTable', 'tbody']).everySubTree('tr');
```

Comment:
This is all more brittle than I would like, even though I added an ID to each table to make it simpler. I think I've spent too much time on this test already, so wondering what you think about making any testing improvements in a separate PR.